### DISCUSSION TITLE
Feature suggestion: use vitepress-sandbox instead of stackblitz

### DISCUSSION THREAD

**Discussion by jerrywu001** [wevm/viem#1360] [Idea / Feature Request]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.1.0

### Current Behavior

It is recommended to use [vitepress-sandbox-plugin](https://vitepress-sandbox.js-bridge.com/get-started/introduction.html) to render demos in MD.

[install docs](https://vitepress-sandbox.js-bridge.com/get-started/install.html)

For example, it is now used by [blocknotejs](https://www.blocknotejs.org/docs/image-toolbar).

Preview screen:

![image](https://github.com/project-nv/night-vision/assets/12525415/63f7ff84-b275-44f5-ae57-c1ffb4528d8a)

Code in md, write:

![image](https://github.com/project-nv/night-vision/assets/12525415/9ec25855-bc4f-4520-9bc5-81b1af713bce)

It supports dark automatic

![image](https://github.com/project-nv/night-vision/assets/12525415/270170db-066e-45eb-97d6-54cbacb19764)

### Expected Behavior

https://vitepress-sandbox.js-bridge.com/get-started/introduction.html

**Comment by jxom** [comment]
Interesting! Will check it out.
